aiogram/dispatcher/event/handler.py

```diff
@@ -19,21 +29,22 @@ class CallableMixin:
     awaitable: bool = field(init=False)
     spec: inspect.FullArgSpec = field(init=False)
 
-    def __post_init__(self):
+    def __post_init__(self) -> None:
         callback = self.callback
         self.awaitable = inspect.isawaitable(callback) or inspect.iscoroutinefunction(callback)
         while hasattr(callback, "__wrapped__"):  # Try to resolve decorated callbacks
-            callback = callback.__wrapped__
+            callback = callback.__wrapped__  # type: ignore
         self.spec = inspect.getfullargspec(callback)
 
-    def _prepare_kwargs(self, kwargs):
+    def _prepare_kwargs(self, kwargs: Dict[str, Any]) -> Dict[str, Any]:
         if self.spec.varkw:
             return kwargs
 
         return {k: v for k, v in kwargs.items() if k in self.spec.args}
 
-    async def call(self, *args, **kwargs):
-        wrapped = partial(self.callback, *args, **self._prepare_kwargs(kwargs))
+    async def call(self, *args: Any, **kwargs: Any) -> Any:
+        # TODO: what we should do if callback is BaseHandler?
```

Comment:
BaseHandler isn't callable so partial can break

Reply:
What can break?

classes is always callable and instances of BaseHandler is awaitable i don't think partial can break something